### DISCUSSION TITLE
Ilaria (she/her) Volunteer @CAT and @SUX

### DISCUSSION THREAD

**Discussion by ilfff** [Green-Software-Foundation/discussions#87] [Introductions]
Hi everyone! 👋
My name is Ilaria, I am an Italian UX and Design Strategist.
I am trying to learn as much as I can about Sustainable Digital Practices in order to adopt them in my everyday and future works. 
I'm already part of ClimateAction.Tech and The Sustainable UX Network Slack communities. 
I am excited about the idea of ​​meeting professionals from different countries and learning from each of you.
Here's my [Linkedin](https://www.linkedin.com/in/ilaria-farina/) feel free to connect.

**Comment by SandrineResponsibleDesigner** [comment]
Hi Ilaria!
Nice to meet a fellow Responsible Designer here :)
Currently working on a Responsible Design Playbook – would like to connect with other practitioners to understand your approach, go-to tools, challenges and successes. 
Here is my [Linkedin](https://www.linkedin.com/in/sandrineherbert/) to connect and [a post on the topic](https://github.com/orgs/Green-Software-Foundation/discussions/90) – maybe the start of a workstream here?

**Reply by pdematos12** [reply]
Hi both. So glad you have joined. I am keen to connect with other designers and UX'ers. I am a User researcher at AVEVA and have recently joined the Patterns and Principles projects to bring a design community around some of those practises.

**Comment by pdematos12** [comment]
Hi @ilfff 

You may want to join this new project for design practitioners.
See sign up details here: https://github.com/Green-Software-Foundation/patterns/discussions/309
